components/fonts/arphic-ukai/history

```diff
@@ -0,0 +1 @@
+system/font/arphic-ukai@0.2.20080216.1-2018.0.0.1 system/font/truetype/arphic-ukai
```

Comment:
Sorry, I am irritated.
From this line I would expect that you want to obsolete system/font/arphic-ukai. But
font.mk sets the default COMPONENT_CLASSIFICATION to System/Font and there is nothing in the Makefile. So why are you creating a new revision just to obsolete the package?
Furthermore, I expect that you'll need to increment the revision in the history file again because the new package revision is already 1 and thus, in order to obsolete it here it must be incremented again.

Reply:
I want to obsolete system/font/arphic-ukai which was published by mistake to enforce the correct FMRI. In this PR system/font/arphic-ukai@0.2.20080216.1-2018.0.0.1 is not published since it is not the correct FMRI, but system/font/truetype/arphic-ukai@0.2.20080216.1-2018.0.0.1 together with the rename rules. The latest revision in the repo is 0. Maybe I misunderstood? You can check that COMPONENT_FONT_TYPE = $(PKGFONTCLS) which is part of the FMRI.

Reply:
Ah, ok. I misread font.mk. You are right.